src/mpyl/cli/commands/build/jenkins.py

```diff
@@ -28,6 +28,7 @@ class JenkinsRunParameters:
     verbose: bool
     follow: bool
     tag: Optional[str] = None
+    tag_target: Target = Target.ACCEPTANCE
```

Comment:
just curious, why are we using Acceptance as default?

Reply:
It's the logical next target to deploy a tag to. I mean, presumably you've tested your PR (so in TEST), so after merging you want to check in acce